src/reference/01-General-Info/90-Changes/45-Migrating-from-sbt-0.13.x.md

```diff
@@ -89,6 +89,18 @@ sbt:hello> Test / fork
 [info] true
 ```
 
+To aid migration a [Scalafix](https://scalacenter.github.io/scalafix/) rewrite rule was created. After setting
+up Scalafix (either locally to a project or user-wide) you should be able to migrate your build with:
+
+```scala
+> sbtfix github:sbt/sbto/v1_1
```

Comment:
Could we hold off the notes about scalafix until it's finalized?
Also typo? "sbt/sbto"

Reply:
Sure (my thinking is these are only 1.x-beta docs, so they can be a little imprecise).

Moved this commit to https://github.com/dwijnand/sbt-website/tree/slashfix, including the typo fix - thanks.